gloo/docs/operator_guide/cert-manager.md

```diff
@@ -0,0 +1,172 @@
+---
+title: "Integrating Gloo and Let's Encrypt with cert-manager"
+weight: 2
+---
+
+This document shows how to secure your ingress traffic using gloo and cert-manager. We will deploy everything
+to minikube. With minor adjustments can be applied to any Kubernetes cluster.
+
+# Pre-requisites
+
+A DNS that your control and supported by cert-manager.
+In this example we used 'test.solo.io' domain that's managed by AWS Route53.
+
+Kubernets cluster (this document was test with minikube and linux, other OSes\clusters should work with proper adjustments).
+
+# Setup
+
+## Setup your DNS
+In this example we used the domain name 'test.solo.io'. We've set an A record for this domain to resolve to the result of `minikube ip` so we can test with minikube.
+
+While you can update your aws dns settings throught the AWS UI, I find performing changes through
+the command line faster. Update the dns record through the aws commandline:
+
+```
+export INGRESS_HOST=$(kubectl get po -l istio=ingressgateway -n istio-system -o 'jsonpath={.items[0].status.hostIP}')
+RECORD=test
+HOSTED_ZONE=solo.io.
```

Comment:
this will obviously only work if you control the solo.io domain.. should we be explicit about that?